src/python/grapl-tests-common/grapl_tests_common/dump_dynamodb.py

```diff
@@ -16,10 +17,10 @@
 def dump_dynamodb() -> None:
     dynamodb: DynamoDBServiceResource = boto3.resource(
```

Comment:
this is something that should definitely use the DynamoDBResourceFactory

Reply:
Good catch!